#
# Copyright (c) 2010, 2018 Oracle and/or its affiliates. All rights reserved.
#
# This program and the accompanying materials are made available under the
# terms of the Eclipse Public License v. 2.0, which is available at
# http://www.eclipse.org/legal/epl-2.0.
#
# This Source Code may also be made available under the following Secondary
# Licenses when the conditions for such availability set forth in the
# Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
# version 2 with the GNU Classpath Exception, which is available at
# https://www.gnu.org/software/classpath/license.html.
#
# SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
#

dsob.delim_not_specified=Delimiter not specified in configuration
dsob.class_not_found=Class name is wrong or classpath is not set for : {0}
dsob.error_instantiating=Error instantiating class : {0}
dsob.access_error=Access error in instantiating class : {0}
dsob.escape_char_not_specified=Escape character not specified in configuration
jdbc.cannot_allocate_connection=Connection could not be allocated because: {0}
jdbc.conn_obj_null=Connection object cannot be null
jdbc.mc_construct_err=The managed connection factory that has created this managed connection is not the same as the managed connection factory returned by the password credential for this managed connection.
jdbc.conn_handle_null=Connection handle cannot be null
jdbc.cannot_get_iso_lvl=The isolation level for this connection could not be retrieved
jdbc.error_in_destroy=Error while destroying resource :
jdbc.mc_not_usable=This Managed Connection is not valid as the physical connection is not usable
jdbc.conn_not_usable=This physical connection is not usable
me.access_denied=Access denied to execute this method : {0}
me.illegal_args=Arguments are wrong for the method :  {0}
me.invalid_param=Parameter wrong for this method : {0}
su.no_passwd_cred=No password credential found
jdbc.cannot_enlist=Cannot enlist this managed connection in this transaction because : {0}
jdbc.cannot_assoc=Cannot associate this managed connection because : {0}
jdbc.error_during_setAutoCommit=SQL Exception : Unable to set AutoCommit property.
jdbc.statement-cache.datastructure.init.failure=Error while creating the Statement cache data structure for type : {0}
jdbc.statement-cache.datastructure.init.failure.exception=Exception : {0}
jdbc.statement-cache.default.datastructure=LRUCacheImpl
jdbc.exc_destroy=Exception while destroying physical connection
